Integrations/integration-PaloAltoNetworks-pandb.yml

```diff
@@ -0,0 +1,464 @@
+description: URL category according to PAN DB
+detaileddescription: |-
+  The integration uses the FW XML API.
+  To obtain an API Key, run the following REST command and copy the key:
+  https://[FWIP]/api/?type=keygen&user=[user]&password=[password]
+
+  For more information, visit the [Palo Alto Networks documentation](https://www.paloaltonetworks.com/documentation).
+configuration:
+- display: Server URL (e.g., https://192.168.0.1)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Port (e.g 443)
+  name: port
+  defaultvalue: "443"
+  type: 0
+  required: false
+- display: API Key
+  name: key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Trust any certificate (unsecure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+script:
+  script: |2-
+
+    ''' IMPORTS '''
+    from datetime import datetime
+    import requests
+    import json
+    import uuid
+    from typing import Dict, List, Any, Optional
+
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    if not demisto.params().get('port'):
+        return_error('Set a port for the instance')
```

Comment:
Set the port parameter to be required, and this verification in the code won't be needed.